From 4993b3436560693941cb5e212b3b7276fb8f4a12 Mon Sep 17 00:00:00 2001
From: Howard Chu <hyc@openldap.org>
Date: Tue, 4 May 2021 10:04:20 +0100
Subject: [PATCH] ITS#9544 fix #9511 patch

Include was erroneously deleted. partially reverts 6cdfc85fc56130628670ca9fe76d6034ddabdd8d
---
 servers/slapd/slapi/plugin.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/servers/slapd/slapi/plugin.c b/servers/slapd/slapi/plugin.c
index 5dee24f9f0..dafeeeb946 100644
--- a/servers/slapd/slapi/plugin.c
+++ b/servers/slapd/slapi/plugin.c
@@ -33,6 +33,8 @@
 #include "slapi.h"
 #include "lutil.h"
 
+#include <ltdl.h>
+
 static int slapi_int_load_plugin( Slapi_PBlock *, const char *, const char *, int, 
 	SLAPI_FUNC *, lt_dlhandle * );
 
-- 
2.20.1

